src/Game.ts

```diff
@@ -341,8 +344,26 @@ export class Game extends EventEmitter<Events> {
             socket.off(`game/${game_id}/move`, on_move);
         });
 
+        if (config.log_game_chat) {
+            socket.send("chat/join", {
+                channel: `game-${game_id}`,
+            });
+            const on_chat = (d) => {
+                // Since there is no explicit tracking of which chats are
+                // "read", we assume anything from before we connected to the
+                // game has already been dealt with.
+                handleChatLine(game_id, d.line, this.startup_timestamp);
+            };
+            socket.on(`game/${game_id}/chat`, on_chat);
+            this.on("disconnecting", () => {
+                socket.off(`game/${game_id}/chat`, on_chat);
+            });
+        }
+
+        trace.log("connecting");
```

Comment:
probably remove this or mention what game we're connecting for if you keep it

Reply:
Whoops, yes, this was a leftover from debugging, will remove!